docs/source/notes/faq.rst

```diff
@@ -18,57 +26,70 @@ please remove the installation directory from your MATLAB path.
     >> savepath % save the new path
     >> delete(CBTDIR,'s') % delete the installation directory
 
-Github
-------
 
-**What do all these labels mean?**
+When running ``git submodule update``, the following error message appears. What should I do?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A comprehensive list of labels and their description for the issues and
-pull requests is given
-`here <https://opencobra.github.io/cobratoolbox/docs/labels.html>`__.
+.. code::
 
-Reconstruction
---------------
+    No submodule mapping found in .gitmodules for path 'external/lusolMex64bit'
+
+remove the cached version of the respective submodule by typing:
+
+.. code::
+
+    git rm --cached external/lusolMex64bit
 
-What does ``DM_reaction`` stand for after running
-``biomassPrecursorCheck(model)``?
+Note: The submodule throwing an error might be different than
+``external/lusolMex64bit``, but the command should work with any submodule.
 
-**Answer**: ``DM_ reactions`` are commonly demand reactions.
 
-Submodules
-----------
+On Windows, MATLAB R2016b crashes with CPLEX 12.7.1. Why?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When running ``git submodule update``, the following error message
-appears:
+When you experience an unexpected crash of MATLAB ``R2016b`` when running:
 
 .. code::
 
-    No submodule mapping found in .gitmodules for path 'external/lusolMex64bit'
+    >> changeCobraSolver('ibm_cplex')
 
-**Solution**: remove the cached version of the respective submodule
+or
 
 .. code::
```

Comment:
`MATLAB` code has to be within a `.. code-block:: matlab` instead of `.. code::`.